src/main/java/uk/gov/hmcts/sptribs/common/service/CcdSupplementaryDataService.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.hmcts.sptribs.common.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.reform.idam.client.models.User;
+import uk.gov.hmcts.sptribs.config.CaseFlagsConfiguration;
+import uk.gov.hmcts.sptribs.idam.IdamService;
+
+import java.util.HashMap;
+import java.util.Map;
+import javax.servlet.http.HttpServletRequest;
+
+import static java.util.Collections.singletonMap;
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+
+@Service
+public class CcdSupplementaryDataService {
+
+    @Autowired
+    private IdamService idamService;
+
+    @Autowired
+    private AuthTokenGenerator authTokenGenerator;
+
+    @Autowired
+    private CoreCaseDataApi coreCaseDataApi;
+
+    @Autowired
+    private CaseFlagsConfiguration caseFlagsConfiguration;
+
+    @Autowired
+    private HttpServletRequest request;
+
+    @Autowired
+    private AuthorisationService authorisationService;
+
+    public void submitSupplementaryDataToCcd(String caseId) {
```

Comment:
Don't think case linking need supplementary data. Remove if not required.

Reply:
Removed this file from PR.